third_party/spdk_defs/nvme_defs.h

```diff
@@ -303,7 +305,7 @@ static_assert(sizeof(SglDescriptor) == 16);
 // https://nvmexpress.org/wp-content/uploads/NVM-Express-1_4-2019.06.10-Ratified.pdf
 struct GenericQueueEntryCmd {
   // dword 0
-  uint16_t opc : 8;   // opcode
+  uint8_t opc : 8;    // opcode
```

Comment:
This field was altered to allow for warning-less opcode casting (opcode is stored as uint8_t).

Reply:
That won't be necessary if `static_cast` is used.